pupil_src/shared_modules/head_pose_tracker/controller/online_controller.py

```diff
@@ -47,6 +47,8 @@ def _on_recent_events(self, events):
             self._calculate_current_markers(events["frame"])
             self._calculate_current_pose(events["frame"])
             self._save_key_markers()
+            events["head_pose"] = self._create_position_events(
+                    self._localization_storage.current_pose)
```

Comment:
Please make sure to run the [black formatter](https://black.readthedocs.io/en/stable/installation_and_usage.html) on your code changes. You could not know that as we haven't set up contribution guidelines yet, but we enforce black formatted python code.

You can e.g. in Visual Studio Code run *Format Document* and then select black as formatter.
Or you run it manually:
```bash
pip install black
black pupil_src\shared_modules\head_pose_tracker\controller\online_controller.py
```